.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,37 @@
+# Summary
+
+Please provide a summary of this issue.
+
+## If You Are Reporting a Bug
+
+If you're reporting a bug, please provide a detailed description of the problem.
+
+## If You Are Submitting a Feature Request
+
+If you're submitting a feature request, please provide as much details on what you'd like to see and 1 or more use cases. The more detail we have, the more quickly and efficiently we can discuss, decide and implement the feature.
+If this is a bug: provide a description of the problem
```

Comment:
Thanks, fixed in https://github.com/arschles/deis-builder/commit/dc1b00d7ac2ff7ac35450fdf367375384a90c3ad